src/GafferUI/AnnotationsGadget.cpp

```diff
@@ -249,6 +249,11 @@ void AnnotationsGadget::doRenderLayer( Layer layer, const Style *style ) const
 	}
 }
 
+int AnnotationsGadget::hasLayer() const
+{
+	return (int)GraphLayer::Overlay;
```

Comment:
Sounds reasonable, but not actually possible to do for an enum, since conversions have to be declared inside the class?  At least, that's what I'm getting from here: https://groups.google.com/a/isocpp.org/g/std-proposals/c/eiN1Xqrt59Y

Reply:
OK, explicit cast it is then.